mmselfsup/models/heads/mae_finetune_head.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Suggest writing `MAEPretrainHead` and `MAEFinetuneHead` into one file named with `mae_head.py`